pkg/validations/commandvalidations/commandvalidations.go

```diff
@@ -0,0 +1,82 @@
+package commandvalidations
+
+import (
+	"context"
+	"fmt"
+	"runtime"
+
+	"github.com/aws/eks-anywhere/pkg/cluster"
+	"github.com/aws/eks-anywhere/pkg/dependencies"
+	"github.com/aws/eks-anywhere/pkg/validations"
+	"github.com/aws/eks-anywhere/pkg/workflows/interfaces"
+)
+
+type ValidateCreateCluster struct {
+	clusterSpec    *cluster.Spec
+	deps           *dependencies.Dependencies
+	cmdValidations interfaces.Validator
+	dockerExec     validations.DockerExecutable
+}
+
+func New(clusterSpec *cluster.Spec, deps *dependencies.Dependencies, cmdValidations interfaces.Validator, dockerExec validations.DockerExecutable) *ValidateCreateCluster {
```

Comment:
Using New as the name for a constructor is only a good idea when the name of the package tells you already what that constructor is building. In this case `commandvalidations.New()` would be telling a lie